tests/scripts/dumpModelRoundTrip.sh

```diff
@@ -22,7 +22,6 @@ ${PODIO_BASE}/tools/podio-dump --dump-edm ${EDM_NAME} ${INPUT_FILE} > ${DUMPED_M
 
 # Regenerate the code via the class generator and the freshly dumped modl
 ${PODIO_BASE}/python/podio_class_generator.py \
-    --clangformat \
```

Comment:
This needs to stay in, because if `clang-format` is detected the original version will be clang-formatted. If no `clang-format` is found, generation will issue a warning about it, but continue.

Reply:
OK. Will re-add it then.